php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesPerformer.java

```diff
@@ -425,28 +432,45 @@ private String createInsertString(final List<UsePart> useParts) {
         } else {
             Collections.sort(useParts);
         }
-        if (!useParts.isEmpty()) {
-            insertString.append(NEW_LINE);
-        }
+
         String indentString = null;
         if (options.preferGroupUses()
                 || options.preferMultipleUseStatementsCombined()) {
             CodeStyle codeStyle = CodeStyle.get(baseDocument);
             indentString = IndentUtils.createIndentString(codeStyle.getIndentSize(), codeStyle.expandTabToSpaces(), codeStyle.getTabSize());
         }
 
-        if (options.preferGroupUses()
+        usesInsertStringHelper.setWrapInNewLinesOnAppend(useParts.size() == 1);
+        if (options.preferGroupUses() && usesInsertStringHelper.canGroupUses()
                 && options.getPhpVersion().compareTo(PhpVersion.PHP_70) >= 0) {
-            insertString.append(createStringForGroupUse(useParts, indentString));
+            createStringForGroupUse(usesInsertStringHelper, useParts, indentString);
         } else if (options.preferMultipleUseStatementsCombined()) {
-            insertString.append(createStringForMultipleUse(useParts, indentString));
+            createStringForMultipleUse(usesInsertStringHelper, useParts, indentString);
         } else {
-            insertString.append(createStringForCommonUse(useParts));
+            createStringForCommonUse(usesInsertStringHelper, useParts);
+        }
+
+        if (!usesInsertStringHelper.isAppendUses()) {
+            try {
+                String replaceString = baseDocument.getText(usesInsertStringHelper.getInitialStartOffset(), usesInsertStringHelper.getStartOffset() - usesInsertStringHelper.getInitialStartOffset());
+                if (replaceString.equals(usesInsertStringHelper.getResultString())) {
+                    usesInsertStringHelper.resetResultString();
+                    return;
+                }
+            } catch (BadLocationException ex) {
+                Exceptions.printStackTrace(ex);
```

Comment:
We can remove `Exceptions.printStackTrace(ex);`.

Reply:
Done